src/__tests__/index.spec.ts

```diff
@@ -1378,10 +1378,15 @@ describe("flatpickr", () => {
       createInstance({ mode: "time" });
       fp.open();
 
-      fp.minuteElement.focus();
+      expect(fp.hourElement).toBeDefined();
+      expect(fp.minuteElement).toBeDefined();
+      expect(fp.amPM).toBeDefined();
+
+      if (!fp.hourElement || !fp.minuteElement || !fp.amPM) return;
+
       simulate(
         "keydown",
-        document.activeElement,
+        fp.minuteElement,
         {
```

Comment:
Using `document.activeElement` is crucial here to test the tabbing flow. If there are potential null errors, just add a `!` assertion

Reply:
Addressed in further commit.

Reply:
I prefer not to use non-null assertion if at all possible.
activeElement checked at each stage of the test although I still believe the last commit is not necessary